tasmota/include/tasmota_template.h

```diff
@@ -2874,11 +2880,11 @@ const mytmplt kModules[] PROGMEM = {
     AGPIO(GPIO_USER),            // 0       IO                  GPIO0, ADC1_CH0,  RTC
     AGPIO(GPIO_USER),            // 1       IO                  GPIO1, ADC1_CH1,  RTC
     AGPIO(GPIO_USER),            // 2       IO                  GPIO2, ADC1_CH2,  RTC
-    AGPIO(GPIO_USER),            // 3       IO                  GPIO3, ADC1_CH3,  RTC 
```

Comment:
Looks like my editor cleaned up some whitespace let me know if this needs to be reverted